StRoot/StBFChain/BigFullChain.h

```diff
@@ -1192,6 +1196,7 @@ Bfc_st BFC[] = { // standard chains
   {"useLDV" ,""  ,"","",""                                   ,"","... uses laserDV database flavor",kFALSE},
   {"useCDV" ,""  ,"","",""                                       ,"","... uses ofl database flavor",kFALSE},
   {"useNewLDV" ,""  ,"","",""                                    ,"","... uses ofl database flavor",kFALSE},
+  {"TFGdbOpt" ,""  ,"","",""                ,"","... uses TFG database flavor for alignemnt tables",kFALSE},
```

Comment:
Does this imply that TFG now has distinct database entries in addition to distinct software? It is unclear to me why we need this.

Reply:
I see this note in the documentation:
"TFGdbOpt to provide a switch between new and old Alignment DB tables."

If this is the purpose, then we should not be using "TFG" in the chain option, nor in the database entries' flavors (production should never depend on anything labeled as developmental). Something more appropriate to "Alignment2024" is more appropriate. And if this is only meant to pair with the new `Alignment2024` chain option, why not directly include the actions as all part of `Alignment2024`?

Reply:
Ignored